cfpb/consumerfinance.gov
# Deprecate YES Transportation Tool

This commit deprecates the Youth Employment Success Transportation Tool currently located [here](https://www.consumerfinance.gov/consumer-tools/educator-tools/resources-youth-employment-programs/transportation-tool/).

It retires the Django view, custom templates, and all associated JavaScript code. A very small amount of JS utility code was being relied on by the related YES Buying a Car Tool [here](https://www.consumerfinance.gov/consumer-tools/educator-tools/resources-youth-employment-programs/buying-a-car/) and so that code has been relocated.

## How to test this PR

1. Python unit tests should pass: `tox`
2. JS unit tests pass (and test the relocated code): `yarn run gulp test:unit`
3. JS linter passes: `yarn run gulp lint`
4. Frontend build works: `./frontend.sh`
5. YES Transpo Tool returns 404 locally: http://localhost:8000/consumer-tools/educator-tools/resources-youth-employment-programs/transportation-tool/
6. YES Buying a Car Tool works locally, including checkboxes and print button at the bottom of the page: http://localhost:8000/consumer-tools/educator-tools/resources-youth-employment-programs/buying-a-car/

## Notes and todos

Question for FEWDs: I wasn't sure where to move the tests for the JS code that got relocated under cfgov/unprocessed/js/routes/on-demand/youth-employment-programs/buying-a-car/. We don't currently have any unit tests under test/js/routes. Does this live elsewhere, or do we just not have any unit test code for routes/on-demand?

Relatedly, our Jest config currently [ignores](https://github.com/cfpb/consumerfinance.gov/blob/81f62d3a85fc91b7d5f2f4a59a6c67006a4f0859/jest.config.js#L20) cfgov/unprocessed/js/routes, which presumably we wouldn't want to if we were actually testing some of that code.

## Checklist
